src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -106,6 +106,27 @@ class ProfileWrapper extends React.Component {
 		);
 	}
 
+  currentGameFnctn(gameId) {
+    window.location = `#/table/${gameId}`;
+  }
+
+	currentGame() {
+
+    if (userInfo.username || gameInfo.publicPlayersState.find(player => player.username === userInfo.userName)) {
+      return (
+        <React.Fragment>
+          <div>
+            <button className="ui primary button currentgame-button" onClick={this.currentGameFnctn}>

```

Comment:
Have you tested this? you call `this.currentGameFctn` here, but above you define it as having an argument of `gameID` which is not included in this onClick.

Reply:
I haven't tested this at all. I'm just getting the code written at first then I'll test afterwards.